src/utils/exceptions.py

```diff
@@ -0,0 +1,17 @@
+import utils.constants
+
+class ATMValidationError(Exception):
+    def __init__(self, code):
+        message = ''
+        if (code == constants.missingFields):
+            message = 'Request is missing one or more of the following fields: lat, lng, rad.'
+        elif(code == constants.invalidFields):
+            message = 'Request contains invalid fields. Lat must be between -90 and 90. Lng must be between -180 and 180.'
+        else:
+            'Unrecognized Error'
```

Comment:
Unsure what happens here if you just have a string
Did you mean to do a `message = 'Unrecognized Error'`?